### DISCUSSION TITLE
Colors

### DISCUSSION THREAD

**Discussion by SylvainMartel** [actualbudget/releases#32] [Feature ideas]
Hi, 
  Some small improvements could be made regarding colors in the budget.
When a category is in the "red", the balance appears in red, but I would also change the balance color when it's positive to either green or blue to make it clear(and black when at zero)

Also, for the Categroy groups, the line are currently pale grey, I would make the contrast bigger(and maybe use a different color than grey) so it's clear that this line is a group.

Having better contrast would help with aging eyes and accessibility :)

**Comment by rft183** [comment]
I agree. The way it is now, it is really easy to miss which categories are overspent and which have money left in them. Adding some color to the balances would be a great quality-of-life improvement.
